tests/Integration/Resources/ProxyProvider/Test_DIRACCAProxyProvider.py

```diff
@@ -12,11 +12,7 @@
 from DIRAC.Core.Security.X509Chain import X509Chain  # pylint: disable=import-error
 from DIRAC.Resources.ProxyProvider.ProxyProviderFactory import ProxyProviderFactory
```

Comment:
what's the difference with the other `Test_DIRACCAProxyProvider`

Reply:
`DIRAC/Resources/ProxyProvider/test/Test_DIRACCAProxyProvider.py` - unit tests, `DIRAC/tests/Integration/Resources/ProxyProvider/Test_DIRACCAProxyProvider.py` - integration test with the main difference being that `CFG`, `ProxyProviderFactory` is used in integration tests to define `ProxyProvider`. Maybe this is over-testing and enough of unit tests.

Reply:
I'll prepare another RP with `ProxyProviderFactory` unit test.